apps/next13/package.json

```diff
@@ -22,18 +22,18 @@
     "@tanstack/react-query": "4.0.10",
     "blitz": "2.1.3",
     "flatted": "3.2.7",
-    "next": "14.2.15",
+    "next": "15.0.1",
     "prisma": "^4.5.0",
-    "react": "18.2.0",
```

Comment:
https://react.dev/blog/2024/12/05/react-19 official 19 released

Reply:
thanks @gengjiawen. will update the PR

Reply:
done